### DISCUSSION TITLE
Frontend / Backend segregation for prod use

### DISCUSSION THREAD

**Discussion by stevenli6186** [dani-garcia/vaultwarden#2008] [Q&A]
How can I separate the database/vault from the frontend?

**Comment by BlackDex** [comment]
I'm not sure what you mean with this. Could you explain what you end goal is, it what it is you are trying to solve?

**Reply by stevenli6186** [reply]
Hi... thanks for your response...

In most of my production services if my service requires a database I use AWS RDS.  Is there a way to use Vaultwarden's front end and deploy  Vaultwarden's backend on separate containers?

The frontend would sit on public subnet for example and the vault/database would sit on a private subnet.

**Reply by BlackDex** [reply]
You are free to use any MySQL and PostgreSQL compatible service where ever you like it to have it running.
As long as Vaultwarden is able to access the database it should work fine.

**Reply by stevenli6186** [reply]
My question was more, where can I find the configs to change the database connection.  I don't even see those configs available in the admin portal.

**Reply by BlackDex** [reply]
See the database section here https://github.com/dani-garcia/vaultwarden/wiki

**Reply by stevenli6186** [reply]
Oh! nevermind I did find it on the Admin portal... where would it be for the config file on the server/container? 

Also, is it possible to build a container without the sqlite database built in?

**Comment by BlackDex** [comment]
You can't change the database via the admin panel.
You should really scroll through the wiki, it has all you need.

https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview
https://github.com/dani-garcia/vaultwarden/wiki/Building-binary

Though, you can build it without sqlite, it's isn't needed, all types of supported database are build in, you can just switch.
